work_with_builds/using-builds.adoc

```diff
@@ -12,8 +12,12 @@ After installing {builds-shortname}, you can create a `buildah` or `source-to-im
 
 include::modules/ob-creating-a-buildah-build.adoc[leveloffset=+1]
 
+include::modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc[leveloffset=+2]
+
```

Comment:
You could include the .Additional Resources section here, for example. The Contributing Guide has guidance on how to do this.